src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,10 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ moviesFromServer }) => (
+  <div className="movies">
+    {moviesFromServer && moviesFromServer.map(movie => (
+      <MovieCard movie={movie} key={movie.imdbId} />
```

Comment:
It's better to destructure the movie object in the map function for better readability. For example, instead of using movie.imdbId, you can use const {imdbId} = movie, and then use imdbId directly.